_posts/2015-09-17-logging-with-Monolog-into-logentries.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Logging with Monlog on LogEntries."
+author: sachit
+modified: 2015-09-17
+tags: [ laravel, logging, logEntries, loggly ]
+---
+
+I am currently working on a [Laravel](http://laravel.com/) project which has taught me a hell lot of stuffs and among those that I have learnt, I'll be shedding a little light on [Logging](), [the importance of it](http://geshan.com.np/blog/2015/08/importance-of-logging-in-your-applications/) and how and where to log.
+
+I'm sure everybody that comes across this post know what logging is. But the importance of it, in a nutshell, is like having a new perspective towards how an application is functioning. My mentor and our project co-ordinator Mr. Geshan Manandhar had said in his [blog](http://geshan.com.np/blog/), "If the dog is a man's best friend, a Log is a developer's best friend". Having said that, I would like to you to checkout the [importance of logging](http://geshan.com.np/blog/2015/08/importance-of-logging-in-your-applications/) on his [blog](http://geshan.com.np/blog/2015/08/importance-of-logging-in-your-applications/) since he is lot better at explaining things like these than I am.
```

Comment:
`this post knows what`

don't mention me as the mentor .... don't put so many links of the same post.